atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -168,12 +169,20 @@ def adjust_for_inherit(self):
                            build_info['nvr'], build_info['id'])
 
         all_compose_ids = set(self.compose_ids)
+        original_compose_ids = deepcopy(all_compose_ids)
         all_compose_ids.update(parent_compose_ids)
         self.all_compose_ids = list(all_compose_ids)
+        for compose_id in all_compose_ids:
```

Comment:
Wouldn't be sufficient to list `parent_compose_ids` ?

Reply:
The parent_compose_ids might be the same as the existing ids, in which case no new compose ids are being added.